schemas/session-replay/common/_webview-supported-common-record-schema.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "session-replay/common/_webview-supported-common-record-schema.json",
+  "title": "WebviewSupportedCommonRecordSchema",
```

Comment:
This new addition will prevent Mobile FS and Incremental S records to have the possibility to contain a `generatedByWebViewId`. Indeed such records are generated by the native apps and are not supported by child webviews.

Reply:
Talking with @bcaudan, he suggested `bridgeId` as naming. This way we are agnostic from whether it’s a webview or an iFrame(in the future) and we reuse the bridge concept we already have. I also think it’s a nice option. Wdyt?

Reply:
> However in the player that has 0 SDK bridge notion, rendering stuff based on a bridgeId will be difficult to understand.

Indeed.

> As in the future, we could also support the replaying of other encapsulated environments such as Iframes it's safer to leverage an agnostic encapsulatedReplayId rather than a webViewId

Not super fan of `encapsulatedReplayId`:

- encapsulated is a bit of a mouthful, maybe nested or embedded could work too
- replay would be player oriented, this structure is already used by other product. 

Looking at your description, maybe something like `nestedEnvId` could work.

Reply:
`nestedEnvId` sounds great to me !